app/src/main/res/values-es/help.xml

```diff
@@ -257,51 +257,51 @@
-        <p>Any node in the list can be bookmarked for later use.
-        Nodes which are not bookmarked will be forgotten.<p>
-        <p>Monerujo will choose the optimal bookmarked node each time you use it.
-        It does this by checking the blockheight (how up-to-date
-        is the node?) as well as the response behaviour (how fast does the node respond to requests?).</p>
-        <p>The list is sorted by these characteristics, so the top node would be the one Monerujo
-        would choose right now. The bottom of the list would show very slow or unavailable nodes.</p>
-        <h2>Add a Node</h2>
-        <p>By touching the &quot;Add Node&quot; button at the bottom, you will be asked to
-        enter the node details in the following dialog.
-        The &quot;Address&quot; is the hostname or IP-address of the node - this is the only
-        mandatory entry.
-        Enter the &quot;Port&quot; if the node runs on a non-default port (e.g. 18089).
-        You can also optionally name the node, so you can identify it easier later on.
-        Some nodes require credentials to use them. Enter the provided username &amp;
-        password in the appropriate fields. Now you can &quot;Test&quot; these setting.
-        The &quot;Test Results&quot; will display the blockheight, response time and actual IP used.
-        The result may also be an error - usually because the hostname provided is
-        not reachable in a sensible amount of time or the credentials are incorrect.
-        Or the hostname/port combination does not point to an actual Monero Node!
-        Once the test passes (no error) - you&apos;re set to press &quot;OK&quot; to save &amp;
-        bookmark this node.</p>
-        <h2>Scan for Nodes</h2>
-        <p>Additionally, you can scan the network for nodes. Monerujo will start
-        scanning the network for Remote Nodes on port 18089. It begins by asking your
-        bookmarked nodes for other peers in the Monero P2P network and then continues
-        by asking those for their peers, and so on. If you have no bookmarked nodes
-        (or they don&apos;t tell us about their peers),
-        Monerujo will go straight to the Monero seed nodes hardcoded into Monero. The
-        scan stops when it finds 10 remote nodes in total.</p>
+        <p>Refresca la lista de nodos al mostrar los marcadores de nodos para permitir a Monerujo
+        elegir el mejor para ti.</p>
+        <h2>¿Qué es un Nodo?h2>
+        <p>Monerujo usa un Nodo Remoto (algunas veces también llamado Daemon) para comunicarse con
+        la red de Monero sin tener que descargar y almacenar una copia de la
+        cadena de bloques completa.<p>
+        <h2>Lista de Nodos</h2>
+        <p>Si la lista está vacía, puedes agregar nuevos nodos manualmente o dejar que Monerujo
+        escanee la red por ti. O ambos.</p>
+        <p>La lista de nodos muestra todos los nodos actualmente conocidos. Adicionalmente, la estampa de tiempo
```

Comment:
Suggestion:

Maybe for **"timestamp"** the words **"marca de tiempo"** could fit. Timestamp means, google; a digital record of the time of occurrence of a particular event.), and "marca de tiempo", refers as the same record in time...

Reply:
It can be added to the terminology then to avoid future mistakes :)

Reply:
sure